packages/postcss-svgo/src/index.js

```diff
@@ -28,6 +28,9 @@ function minify(decl, opts, postcssResult) {
       svg = Buffer.from(base64String, 'base64').toString('utf8');
       isBase64 = true;
     } else {
+      if (!dataURI.test(value)) {
```

Comment:
If the value is not Base64-encoded, the code did not check whether the value was an SVG.